### PR TITLE
feat: add subscription for account deletion to send confirmation email

### PR DESCRIPTION
**Pull request by ranim-n**

### CODE REVIEW

File: ngsild-payloads/accountDeletionSubscription.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+	"id": "urn:ngsi-ld:Subscription:AccountDeletion",
```
Comment (bobeal) [inline ngsild-payloads/accountDeletionSubscription.jsonld:2]: others use userProfile, not account, let's be consistent. so ... `UserProfileDeletionNotification`?

Comment (junjou) [reply ngsild-payloads/accountDeletionSubscription.jsonld:2]: What about: `urn:ngsi-ld:Subscription:UserProfileDeletion`

**Review by junjou — APPROVED** [review]

File: ngsild-payloads/accountDeletionSubscription.jsonld (hunk 1 above)
Comment (ranim-n) [reply ngsild-payloads/accountDeletionSubscription.jsonld:2]: `accountDeletion` references more the general action done by the user which will trigger sending the notifcation and therefore the email (the email is to confirm his account deletion after all). `userProfileDeletion` references the entity being deleted, independantly of which action triggered the deletion.

Comment (bobeal) [reply ngsild-payloads/accountDeletionSubscription.jsonld:2]: if we add another action to be done when an user deletes its account, how will you call it then?

Comment (ranim-n) [reply ngsild-payloads/accountDeletionSubscription.jsonld:2]: we can choose a name based on the action. And we can do the same here too. `UserProfileDeletionNotification` is just as generic as `AccountDeletion` and `userProfileDeletion `.  So maybe something like : `urn:ngsi-ld:Subscription:UserProfileDeletion:EmailNotification` or `urn:ngsi-ld:Subscription:AccountDeletion:EmailNotification` ?

Comment (bobeal) [reply ngsild-payloads/accountDeletionSubscription.jsonld:2]: `urn:ngsi-ld:Subscription:UserProfileDeletion:EmailNotification`

(and I guess the other one should be renamed for consistency)

**Review by bobeal — APPROVED** [review]

**Pull request merged**